### PR TITLE
fix cjs export

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

**Review by dios-david — APPROVED** [review]
Thanks a lot 🎉

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		"bench:v8": "BABEL_ENV=test microbundle benchmarks/index.js -f modern --alias benchmarkjs-pretty=benchmarks/lib/benchmark-lite.js --external none --target node --no-compress --no-sourcemap --raw -o benchmarks/.v8.mjs && v8 --module benchmarks/.v8.mjs",
 		"build": "npm run -s transpile && npm run -s transpile:jsx && npm run -s copy-typescript-definition",
 		"postbuild": "node ./config/node-13-exports.js && node ./config/node-commonjs.js",
-		"transpile": "microbundle src/index.js -f es,umd --target web --external preact",
+		"transpile": "microbundle src/index.js -f es,cjs --target web --external preact",
 		"transpile:jsx": "microbundle src/jsx.js -o dist/jsx.js --target web --external preact && microbundle dist/jsx.js -o dist/jsx.js -f cjs --external preact",
```
Comment (andrewiggins) [inline package.json:32]: Does this mean we are no longer producing a build that can load in a browser? What's the bug here?

Comment (JoviDeCroock) [reply package.json:32]: It's using umd globals and it seems like with the named export removed that gives some issues. Can look deeper into it though

**Comment by JoviDeCroock** [comment]
@andrewiggins added back umd and streamlined export maps with the preact ones

**Review by andrewiggins — APPROVED** [review]

**Pull request merged**